common/assets/styles/root.scss

```diff
@@ -265,4 +275,4 @@
 
 .MuiTab-root:active {
   --active-tint: inherit;
-}
+}
```

Comment:
:green_circle: No new line at the end of the file

Reply:
il faut un newline ou pas du coup ? Mon linter ne veut pas que j'en mette

Reply:
:shrug: